app/controllers/icon_redirects_controller.rb

```diff
@@ -3,7 +3,7 @@ def show
     redirect_to view_context.asset_path(request.path.to_s[1..-1]), status: 301
   end
 
-  def apple_60_60
-    redirect_to view_context.asset_path('apple-touch-icon-60x60.png'), status: 301
+  def apple_touch_icon
+    redirect_to view_context.asset_path('apple-touch-icon.png'), status: 301
```

Comment:
The behaviour of this is now the same as the behaviour of `show` (e.g. `/apple-touch-icon.png` becomes a redirect to `asset/path/to/apple-touch-icon.png`) maybe we should get rid of it?